odxtools/diagcodedtype.py

```diff
@@ -3,7 +3,10 @@
 from dataclasses import dataclass
 from typing import TYPE_CHECKING, Any, Dict, Literal, Optional, Tuple, Union
 
-import bitstruct
+try:
+    import bitstruct.c as bitstruct
```

Comment:
Are you sure you want to use `bitstruct.c` ? There are limitations
* Integers and booleans must be 64 bits or less. => ODX limit integers to 64 bits?
* Text and raw must be a multiple of 8 bits. => should be fine
* Bit endianness and byte order are not yet supported. => matters in ODX

See https://github.com/qchateau/cbitstruct as alternative, limitations there sound a bit more realistic for ODX use cases

Reply:
> Are you sure you want to use bitstruct.c ?

pretty much.

> Integers and booleans must be 64 bits or less. => ODX limit integers to 64 bits?

I think it even limits it to 32 bits, see section 7.1.3 (ASAM version of the document).

> Bit endianness and byte order are not yet supported.

we do (and always did) the byte order handling manually. (byte order specifiers are unsupported, i.e., everything i assumed to be big endian by `bitstruct.c`, IIRC.)

Reply:
> See https://github.com/qchateau/cbitstruct as alternative, limitations there sound a bit more realistic for ODX use cases

it seems like this does not support byte fields longer than 64 bits. (if the README is correct...)